amazon-ebs: Crash on step_ami_region_copy

Another one of these, haven't check `1.0` branch but I expect it to be there too.

```
==> amazon-ebs: Copying AMI (ami-398ba05f) to other regions...
    amazon-ebs: Copying to: eu-central-1
    amazon-ebs: Waiting for all copies to complete...
Build 'amazon-ebs' errored: unexpected EOF

==> Some builds didn't complete successfully and had errors:
--> amazon-ebs: unexpected EOF

==> Builds finished but no artifacts were created.
panic: runtime error: invalid memory address or nil pointer dereference
2017/03/07 20:14:56 packer: [signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x1728021]
2017/03/07 20:14:56 packer:
2017/03/07 20:14:56 packer: goroutine 166 [running]:
2017/03/07 20:14:56 packer: github.com/mitchellh/packer/builder/amazon/common.amiRegionCopy(0x2b33160, 0xc420315bc0, 0xc420222040, 0xc4201335c0, 0x16, 0xc420456100, 0xc, 0xc4202d6080, 0xc, 0xc4202d6460, ...)
2017/03/07 20:14:56 packer:     /Users/rickard/src/go-code/src/github.com/mitchellh/packer/builder/amazon/common/step_ami_region_copy.go:132 +0x851
2017/03/07 20:14:56 packer: github.com/mitchellh/packer/builder/amazon/common.(*StepAMIRegionCopy).Run.func1(0xc420456e20, 0x2b33160, 0xc420315bc0, 0xc420315cb0, 0xc420456100, 0xc, 0xc420130190, 0xc420456e08, 0xc420314b40, 0xc42035f980, ...)
2017/03/07 20:14:56 packer:     /Users/rickard/src/go-code/src/github.com/mitchellh/packer/builder/amazon/common/step_ami_region_copy.go:50 +0xf3
2017/03/07 20:14:56 packer: created by github.com/mitchellh/packer/builder/amazon/common.(*StepAMIRegionCopy).Run
2017/03/07 20:14:56 packer:     /Users/rickard/src/go-code/src/github.com/mitchellh/packer/builder/amazon/common/step_ami_region_copy.go:59 +0x567
2017/03/07 20:14:56 ui error: Build 'amazon-ebs' errored: unexpected EOF
```

- Packer v1.0.rc1 (0bd04f7babcfdd802c8c1632432d38b2f9021696+CHANGES)

```json
{
  "builders": [
    {
      "type": "amazon-ebs",
      "region": "eu-west-1",
      "ami_regions": [ "eu-central-1" ],
      "source_ami": "ami-25acd752",
      "instance_type": "t1.micro",
      "ssh_username": "ec2-user",
      "ssh_pty": "true",
      "ami_name": "packer-demo-{{timestamp}}",
      "encrypt_boot": false,
      "tags": {
        "OS_Version": "CentOS",
        "Release": "Latest"
      },
      "ami_block_device_mappings": [
        {
          "device_name": "/dev/sdb",
          "volume_size": 2
        }
      ]
    }
  ]
}
```

--- a/builder/amazon/common/step_ami_region_copy.go
+++ b/builder/amazon/common/step_ami_region_copy.go
@@ -2,7 +2,6 @@ package common
 
 import (
 	"fmt"
-
 	"sync"
 
 	"github.com/aws/aws-sdk-go/aws"
@@ -128,7 +127,7 @@ func amiRegionCopy(state multistep.StateBag, config *AccessConfig, name string, 
 	}
 
 	for _, blockDeviceMapping := range describeImageResp.Images[0].BlockDeviceMappings {
-		if blockDeviceMapping.Ebs != nil {
+		if blockDeviceMapping.Ebs != nil && blockDeviceMapping.Ebs.SnapshotId != nil {
 			snapshotIds = append(snapshotIds, *blockDeviceMapping.Ebs.SnapshotId)
 		}
 	}

